website/docs/r/independent_disk.html.markdown

```diff
@@ -48,7 +47,72 @@ The following arguments are supported:
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
 * `name` - (Required) Disk name
-* `size` - (Required) - Size of disk in MB
-* `bus_type` - (Optional) - Disk bus type. Values can be: IDE, SCSI, SATA 
-* `bus_sub_type` - (Optional) - Disk bus subtype. Values can be: "IDE" for IDE. buslogic, lsilogic, lsilogicsas, VirtualSCSI for SCSI and ahci for SATA
-* `storage_profile` - (Optional) - The name of storage profile where disk will be created
\ No newline at end of file
+* `size` - (Optional; **Deprecated**) Size of disk in MB. On read this values isn't updated.
+* `size_in_bytes` - (Optional; *v2.5+*) Size in bytes
+* `bus_type` - (Optional) Disk bus type. Values can be: IDE, SCSI, SATA 
+* `bus_sub_type` - (Optional) Disk bus subtype. Values can be: "IDE" for IDE. buslogic, lsilogic, lsilogicsas, VirtualSCSI for SCSI and ahci for SATA
+* `storage_profile` - (Optional) The name of storage profile where disk will be created
+
+## Attribute reference
+
+Supported in provider *v2.5+*
+
+* `iops` - (Computed) IOPS request for the created disk
+* `owner_name` - (Computed) The owner name of the disk
+* `datastore_name` - (Computed) Data store name. Readable only for system user.
+* `is_attached` - (Computed) True if the disk is already attached
+
+## Importing
+
+Supported in provider *v2.5+*
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing independent disk can be [imported][docs-import] into this resource via supplying its path.
+The path for this resource is made of org-name.vdc-name.disk-id
+For example, using this structure, representing a independent disk that was **not** created using Terraform:
+
+```hcl
+resource "vcd_independent_disk" "tf-myDisk" {
+  vdc     = "my-vdc"
+  name    = "my-disk"
+}
+```
+
+You can import such independent disk into terraform state using this command
+
+```
+terraform import vcd_independent_disk.tf-myDisk org-name.vdc-name.my-disk-id
+```
+
+[docs-import]:https://www.terraform.io/docs/import/
+
+After importing, if you run `terraform plan` you will see the rest of the values and modify the script accordingly for
+further operations.
+
+### Listing independent disk IDs
+
+If you want to list IDs there is a special command **`terraform import vcd_independent_disk.imported list@org-name.vdc-name.my-independent-disk-name`**
+where `org-name` is the organization used, `vdc-name` is vDC name and `my-independent-disk-name`
+is independent disk name. The output for this command should look similar to below one:
+
+```shell
+$ terraform import vcd_independent_disk.imported list@org-name.vdc-name.my-independent-disk-name
+vcd_independent_disk.Disk_import: Importing from ID "list@org-name.vdc-name.my-independent-disk-name"...
+Retrieving all disks by name
+No	ID							                        Name	    Description	Size
```

Comment:
The table looks to have a drift when looking at website

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/15804230/67364360-1513a380-f578-11e9-9c18-d3d0836fbe2d.png">

Reply:
fixed